src/app.ts

```diff
@@ -7,13 +7,12 @@ var game = require('./controllers/game');
 
 const port = 3000;
 
-
 app.use('/config', config);
 app.use('/board', board);
 app.use('/game', game);
 
 const server = app.listen(port, function() {
-    var host = server.address().address
-    var port = server.address().port
-    console.log("Example app listening at http://%s:%s", host, port)
-})
\ No newline at end of file
+  var host = server.address().address;
+  var port = server.address().port;
+  console.log('Example app listening at http://%s:%s', host, port);
```

Comment:
could probably update the console log now that it's so close! :P

Reply:
Updated message